dwa_local_planner/src/dwa_planner_ros.cpp

```diff
@@ -157,7 +159,7 @@ namespace dwa_local_planner {
     return dp_->setPlan(orig_global_plan);
   }
 
-  bool DWAPlannerROS::isGoalReached() {
+  bool DWAPlannerROS::isGoalReached(double xy_tolerance, double yaw_tolerance) {
```

Comment:
if the arguments are unused, you could just omit the names, i.e. `isGoalReached(double, double)`